addons/account_edi_ubl_cii/data/cii_22_templates.xml

```diff
@@ -197,6 +208,7 @@
                             <t t-call="account_edi_ubl_cii.account_invoice_partner_facturx_export_22">
                                 <t t-set="partner"
                                    t-value="ship_to_trade_party"/>
+                                   <t t-set="hide_dtc" t-value="'DE' == record.company_id.partner_id.commercial_partner_id.country_id.code"/>
```

Comment:
I would prefer if this condition was based on the XML format.
Could you use the value of `document_context_id` instead? It won't look nicer, but it would make more sense as the fact that it should be there or not is based on this.

Reply:
Thanks for the feedback.